hrpc/mutate.go

```diff
@@ -376,6 +409,18 @@ func (m *Mutate) serializeWithReflect() (*pb.MutateRequest, error) {
 	if m.timestamp != MaxTimestamp {
 		mProto.Timestamp = &m.timestamp
 	}
+
+	if m.ttl != nil && len(m.ttl) > 0 {
+		if mProto.Attribute == nil {
+			mProto.Attribute = []*pb.NameBytesPair{}
```

Comment:
no need for this also, if mProto.Attribute is nil, append will create a new slice.